Fix build issues

### Steps

1. Try to build the plugin

### Result

FAILURE: Build failed with an exception.

### Expected Result

Build should be successful.

### IDE Details

N/A

### Stack Trace

```
* What went wrong:
Could not determine the dependencies of task ':generateLexerMtl'.
> Could not resolve all files for configuration ':compileClasspath'.
   > Could not download jogl-all-main-2.5.0.jar (org.jogamp.jogl:jogl-all-main:2.5.0)
      > Could not get resource 'https://www.jogamp.org/deployment/maven/org/jogamp/jogl/jogl-all-main/2.5.0/jogl-all-main-2.5.0.jar'.
         > Could not HEAD 'https://www.jogamp.org/deployment/maven/org/jogamp/jogl/jogl-all-main/2.5.0/jogl-all-main-2.5.0.jar'.
            > Certificate for <www.jogamp.org> doesn't match any of the subject alternative names: [jogamp.org, mail.jogamp.org]
   > Could not download gluegen-rt-main-2.5.0.jar (org.jogamp.gluegen:gluegen-rt-main:2.5.0)
      > Could not get resource 'https://www.jogamp.org/deployment/maven/org/jogamp/gluegen/gluegen-rt-main/2.5.0/gluegen-rt-main-2.5.0.jar'.
         > Could not HEAD 'https://www.jogamp.org/deployment/maven/org/jogamp/gluegen/gluegen-rt-main/2.5.0/gluegen-rt-main-2.5.0.jar'.
            > Certificate for <www.jogamp.org> doesn't match any of the subject alternative names: [jogamp.org, mail.jogamp.org]
   > Could not download jogl-all-2.5.0.jar (org.jogamp.jogl:jogl-all:2.5.0)
      > Could not get resource 'https://www.jogamp.org/deployment/maven/org/jogamp/jogl/jogl-all/2.5.0/jogl-all-2.5.0.jar'.
         > Could not HEAD 'https://www.jogamp.org/deployment/maven/org/jogamp/jogl/jogl-all/2.5.0/jogl-all-2.5.0.jar'.
            > Certificate for <www.jogamp.org> doesn't match any of the subject alternative names: [jogamp.org, mail.jogamp.org]
   > Could not download jogl-all-2.5.0-natives-android-aarch64.jar (org.jogamp.jogl:jogl-all:2.5.0)
      > Could not get resource 'https://www.jogamp.org/deployment/maven/org/jogamp/jogl/jogl-all/2.5.0/jogl-all-2.5.0-natives-android-aarch64.jar'.
         > Could not HEAD 'https://www.jogamp.org/deployment/maven/org/jogamp/jogl/jogl-all/2.5.0/jogl-all-2.5.0-natives-android-aarch64.jar'.
            > Certificate for <www.jogamp.org> doesn't match any of the subject alternative names: [jogamp.org, mail.jogamp.org]
   > Could not download jogl-all-2.5.0-natives-linux-amd64.jar (org.jogamp.jogl:jogl-all:2.5.0)
      > Could not get resource 'https://www.jogamp.org/deployment/maven/org/jogamp/jogl/jogl-all/2.5.0/jogl-all-2.5.0-natives-linux-amd64.jar'.
         > Could not HEAD 'https://www.jogamp.org/deployment/maven/org/jogamp/jogl/jogl-all/2.5.0/jogl-all-2.5.0-natives-linux-amd64.jar'.
            > Certificate for <www.jogamp.org> doesn't match any of the subject alternative names: [jogamp.org, mail.jogamp.org]
   > Could not download jogl-all-2.5.0-natives-linux-armv6hf.jar (org.jogamp.jogl:jogl-all:2.5.0)
      > Could not get resource 'https://www.jogamp.org/deployment/maven/org/jogamp/jogl/jogl-all/2.5.0/jogl-all-2.5.0-natives-linux-armv6hf.jar'.
         > Could not HEAD 'https://www.jogamp.org/deployment/maven/org/jogamp/jogl/jogl-all/2.5.0/jogl-all-2.5.0-natives-linux-armv6hf.jar'.
            > Certificate for <www.jogamp.org> doesn't match any of the subject alternative names: [jogamp.org, mail.jogamp.org]
   > Could not download jogl-all-2.5.0-natives-linux-aarch64.jar (org.jogamp.jogl:jogl-all:2.5.0)
      > Could not get resource 'https://www.jogamp.org/deployment/maven/org/jogamp/jogl/jogl-all/2.5.0/jogl-all-2.5.0-natives-linux-aarch64.jar'.
         > Could not HEAD 'https://www.jogamp.org/deployment/maven/org/jogamp/jogl/jogl-all/2.5.0/jogl-all-2.5.0-natives-linux-aarch64.jar'.
            > Certificate for <www.jogamp.org> doesn't match any of the subject alternative names: [jogamp.org, mail.jogamp.org]
   > Could not download jogl-all-2.5.0-natives-macosx-universal.jar (org.jogamp.jogl:jogl-all:2.5.0)
      > Could not get resource 'https://www.jogamp.org/deployment/maven/org/jogamp/jogl/jogl-all/2.5.0/jogl-all-2.5.0-natives-macosx-universal.jar'.
         > Could not HEAD 'https://www.jogamp.org/deployment/maven/org/jogamp/jogl/jogl-all/2.5.0/jogl-all-2.5.0-natives-macosx-universal.jar'.
            > Certificate for <www.jogamp.org> doesn't match any of the subject alternative names: [jogamp.org, mail.jogamp.org]
   > Could not download jogl-all-2.5.0-natives-windows-amd64.jar (org.jogamp.jogl:jogl-all:2.5.0)
      > Could not get resource 'https://www.jogamp.org/deployment/maven/org/jogamp/jogl/jogl-all/2.5.0/jogl-all-2.5.0-natives-windows-amd64.jar'.
         > Could not HEAD 'https://www.jogamp.org/deployment/maven/org/jogamp/jogl/jogl-all/2.5.0/jogl-all-2.5.0-natives-windows-amd64.jar'.
            > Certificate for <www.jogamp.org> doesn't match any of the subject alternative names: [jogamp.org, mail.jogamp.org]
   > Could not download gluegen-rt-2.5.0.jar (org.jogamp.gluegen:gluegen-rt:2.5.0)
      > Could not get resource 'https://www.jogamp.org/deployment/maven/org/jogamp/gluegen/gluegen-rt/2.5.0/gluegen-rt-2.5.0.jar'.
         > Could not HEAD 'https://www.jogamp.org/deployment/maven/org/jogamp/gluegen/gluegen-rt/2.5.0/gluegen-rt-2.5.0.jar'.
            > Certificate for <www.jogamp.org> doesn't match any of the subject alternative names: [jogamp.org, mail.jogamp.org]
   > Could not download gluegen-rt-2.5.0-natives-android-aarch64.jar (org.jogamp.gluegen:gluegen-rt:2.5.0)
      > Could not get resource 'https://www.jogamp.org/deployment/maven/org/jogamp/gluegen/gluegen-rt/2.5.0/gluegen-rt-2.5.0-natives-android-aarch64.jar'.
         > Could not HEAD 'https://www.jogamp.org/deployment/maven/org/jogamp/gluegen/gluegen-rt/2.5.0/gluegen-rt-2.5.0-natives-android-aarch64.jar'.
            > Certificate for <www.jogamp.org> doesn't match any of the subject alternative names: [jogamp.org, mail.jogamp.org]
   > Could not download gluegen-rt-2.5.0-natives-linux-amd64.jar (org.jogamp.gluegen:gluegen-rt:2.5.0)
      > Could not get resource 'https://www.jogamp.org/deployment/maven/org/jogamp/gluegen/gluegen-rt/2.5.0/gluegen-rt-2.5.0-natives-linux-amd64.jar'.
         > Could not HEAD 'https://www.jogamp.org/deployment/maven/org/jogamp/gluegen/gluegen-rt/2.5.0/gluegen-rt-2.5.0-natives-linux-amd64.jar'.
            > Certificate for <www.jogamp.org> doesn't match any of the subject alternative names: [jogamp.org, mail.jogamp.org]
   > Could not download gluegen-rt-2.5.0-natives-linux-armv6hf.jar (org.jogamp.gluegen:gluegen-rt:2.5.0)
      > Could not get resource 'https://www.jogamp.org/deployment/maven/org/jogamp/gluegen/gluegen-rt/2.5.0/gluegen-rt-2.5.0-natives-linux-armv6hf.jar'.
         > Could not HEAD 'https://www.jogamp.org/deployment/maven/org/jogamp/gluegen/gluegen-rt/2.5.0/gluegen-rt-2.5.0-natives-linux-armv6hf.jar'.
            > Certificate for <www.jogamp.org> doesn't match any of the subject alternative names: [jogamp.org, mail.jogamp.org]
   > Could not download gluegen-rt-2.5.0-natives-linux-aarch64.jar (org.jogamp.gluegen:gluegen-rt:2.5.0)
      > Could not get resource 'https://www.jogamp.org/deployment/maven/org/jogamp/gluegen/gluegen-rt/2.5.0/gluegen-rt-2.5.0-natives-linux-aarch64.jar'.
         > Could not HEAD 'https://www.jogamp.org/deployment/maven/org/jogamp/gluegen/gluegen-rt/2.5.0/gluegen-rt-2.5.0-natives-linux-aarch64.jar'.
            > Certificate for <www.jogamp.org> doesn't match any of the subject alternative names: [jogamp.org, mail.jogamp.org]
   > Could not download gluegen-rt-2.5.0-natives-macosx-universal.jar (org.jogamp.gluegen:gluegen-rt:2.5.0)
      > Could not get resource 'https://www.jogamp.org/deployment/maven/org/jogamp/gluegen/gluegen-rt/2.5.0/gluegen-rt-2.5.0-natives-macosx-universal.jar'.
         > Could not HEAD 'https://www.jogamp.org/deployment/maven/org/jogamp/gluegen/gluegen-rt/2.5.0/gluegen-rt-2.5.0-natives-macosx-universal.jar'.
            > Certificate for <www.jogamp.org> doesn't match any of the subject alternative names: [jogamp.org, mail.jogamp.org]
   > Could not download gluegen-rt-2.5.0-natives-windows-amd64.jar (org.jogamp.gluegen:gluegen-rt:2.5.0)
      > Could not get resource 'https://www.jogamp.org/deployment/maven/org/jogamp/gluegen/gluegen-rt/2.5.0/gluegen-rt-2.5.0-natives-windows-amd64.jar'.
         > Could not HEAD 'https://www.jogamp.org/deployment/maven/org/jogamp/gluegen/gluegen-rt/2.5.0/gluegen-rt-2.5.0-natives-windows-amd64.jar'.
            > Certificate for <www.jogamp.org> doesn't match any of the subject alternative names: [jogamp.org, mail.jogamp.org]
```

--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -35,7 +35,7 @@ repositories {
     }
     maven {
         name = "JogAmp"
-        url = uri(path = "https://www.jogamp.org/deployment/maven/")
+        url = uri(path = "https://jogamp.org/deployment/maven/")
     }
 }
 

